### PR TITLE
[Alexa Adapter] Add unit tests for AlexaRequestMapper class

### PR DESCRIPTION
**Pull request by sw-joelmut**
## Description
This PR adds tests for the [AlexaRequestMapper](https://github.com/BotBuilderCommunity/botbuilder-community-dotnet/blob/develop/libraries/Bot.Builder.Community.Adapters.Alexa.Core/AlexaRequestMapper.cs#L22) class increasing its code coverage from **76%** to **99%**.

### Specific Changes
- Updated the [AlexaRequestMapperTests](https://github.com/BotBuilderCommunity/botbuilder-community-dotnet/blob/develop/tests/Bot.Builder.Community.Adapters.Alexa.Tests/AlexaRequestMapperTests.cs#L25) file with new **xUnit** tests.

## Testing
The following images show the new tests passing and the resulting code coverage.
image
image

### CODE REVIEW

File: tests/Bot.Builder.Community.Adapters.Alexa.Tests/AlexaRequestMapperTests.cs [hunk 1]
```diff
@@ -373,29 +388,90 @@ public void MergeActivitiesWithEndOfConversationOnly()
...
+        [Fact]
+        public void MergeActivitiesWithWhitespacesShouldReturnEmpty()
+        {
+            var alexaAdapter = new AlexaRequestMapper();
+
+            var activity = new Activity
+            {
+                Type = ActivityTypes.Message,
+                Text = "   "
+
```
Comment (ceciliaavila) [inline tests/Bot.Builder.Community.Adapters.Alexa.Tests/AlexaRequestMapperTests.cs]: Please remove this extra line.

File: tests/Bot.Builder.Community.Adapters.Alexa.Tests/AlexaRequestMapperTests.cs [hunk 2]
```diff
@@ -373,29 +388,90 @@ public void MergeActivitiesWithEndOfConversationOnly()
...
+        public void MergeActivitiesWithTextFormatTypePlainShouldReturnValue()
+        {
+            var alexaAdapter = new AlexaRequestMapper();
+
+            var activity = new Activity
+            {
+                Type = ActivityTypes.Message,
+                Text = "</>",
+                TextFormat = TextFormatTypes.Plain
+
```
Comment (ceciliaavila) [inline tests/Bot.Builder.Community.Adapters.Alexa.Tests/AlexaRequestMapperTests.cs]: Same in here.

**Review by ceciliaavila — CHANGES_REQUESTED** [review]

**Review by ceciliaavila — APPROVED** [review]

File: tests/Bot.Builder.Community.Adapters.Alexa.Tests/AlexaRequestMapperTests.cs (hunk 1 above)
Comment (sw-joelmut) [reply tests/Bot.Builder.Community.Adapters.Alexa.Tests/AlexaRequestMapperTests.cs]: Those slippery empty lines, 👍.

**Review by santgr11 — APPROVED** [review]

**Review by ian-whiteout — CHANGES_REQUESTED** [review]
Inconsistent use of implicit/explicit constructor arguments.
Examples:
`new User()`
`new Intent`

**Comment by ceciliaavila** [comment]
> Inconsistent use of implicit/explicit constructor arguments.
> Examples:
> `new User()`
> `new Intent`

**Comment by sw-joelmut** [comment]
> 
> 
> Inconsistent use of implicit/explicit constructor arguments.
> Examples:
> `new User()`
> `new Intent`

It looks good to me, since we remove the explicit `()` when there are properties defined on the class instance, otherwise it should go the `()`. Maybe @ceciliaavila could provide more information on this, thanks.

**Comment by ceciliaavila** [comment]
> > Inconsistent use of implicit/explicit constructor arguments.
> > Examples:
> > `new User()`
> > `new Intent`
> 
> It looks good to me, since we remove the explicit `()` when there are properties defined on the class instance, otherwise it should go the `()`. Maybe @ceciliaavila could provide more information on this, thanks.

Yes, Resharper suggests removing the `()` when you assign values to the properties in the instance.
image

**Review by ian-whiteout — APPROVED** [review]

**Pull request closed without merging**